CONTRIBUTING.rst

```diff
@@ -154,13 +159,14 @@ If you're happy with the result, do a **force** push (since you're rewriting his
 See also: http://www.ericbmerritt.com/2011/09/21/commit-hygiene-and-git.html
 
 
-.. _CLA: https://crate.io/community/contribute/cla/
-.. _Crate.io: http://crate.io/
+.. _ICLA: https://app.hellosign.com/s/LP1Ul5Vj
+.. _CCLA: https://app.hellosign.com/s/b59e3c0a
+.. _Crate.io: http://cratdb.com
```

Comment:
I think we can simply remove this now, (it also has a typo, `e` is missing).

Reply:
Yes, fine for me too to remove it. Do I need to do it, or can you?

Reply:
dit it myself, no worries, thx for this update!